.github/workflows/windows.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Windows release
+
+on:
+  push:
+    branches: [ "main", "ci-*"]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-windows-release:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.12']
+      fail-fast: true
+
+    steps:
+    - uses: actions/checkout@v4.1.2
+      with:
+        fetch-depth: 0
+
+    - name: Get short commit ref
+      run: |
+        version=$(
+            git describe --tags --dirty \
+            | sed -e 's/^v//' -e 's/\([^-]*-g\)/r\1/;s/-/./g'
+          )
```

Comment:
As far as I can tell, this makes something like `0.13.r372.gSHORTSHA`, which isn't a valid Python [version specifier](https://packaging.python.org/en/latest/specifications/version-specifiers/). Maybe something like
```
git describe --tags --dirty | sed -e 's/^v//' -e 's/\([^-]*-g\)/dev\1/;s/-g.*//g'
``` 
which sets the version to `0.13.dev372`?

Reply:
TIL! Sure!

Reply:
addressed in https://github.com/papis/papis/pull/795/commits/5b0de9ffb22f7d5857b26b8571c6e9b391a90bd9

Reply:
Thanks!